scio-core/src/main/scala/com/spotify/scio/coders/instances/AvroCoders.scala

```diff
@@ -125,8 +126,12 @@ trait AvroCoders {
     Coder.beam(new SlowGenericRecordCoder)
 
   import org.apache.avro.specific.SpecificRecordBase
-  implicit def genAvro[T <: SpecificRecordBase]: Coder[T] =
-    macro AvroCoderMacros.staticInvokeCoder[T]
+  implicit def genAvro[T <: SpecificRecordBase: ClassTag]: Coder[T] = {
```

Comment:
Is `SpecificRecord` type applicable here?

Reply:
Do you mean instead of `SpecificRecordBase` ?
It should work. We can probably relax our type bound everywhere

Reply:
Yes, and it remains backward compatible